eigen3/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+load(
+    "//ign_bazel:build_defs.bzl",
+    "IGNITION_ROOT",
+    "IGNITION_VISIBILITY",
+)
+
+package(
+    default_visibility = IGNITION_VISIBILITY,
+    features = [
+        "-layering_check",
+    ],
+)
+
+licenses(["notice"])
+
+exports_files(["LICENSE"])
```

Comment:
nit: the convention we use is that licenses in each BUILD file while exports_files only in the top-level BUILD file.

Reply:
Removed in e7a37b4